genie-agent/build.gradle

```diff
@@ -53,7 +61,6 @@ dependencies {
      *******************************/
 
     implementation("io.grpc:grpc-netty")
-    implementation("io.awspring.cloud:spring-cloud-starter-aws")
```

Comment:
you sure this is safe to drop? for reference, Spring Cloud 3 starter name is `io.awspring.cloud:spring-cloud-aws-starter`